src/GSF.CarbonAware/src/Handlers/ForecastHandler.cs

```diff
@@ -0,0 +1,41 @@
+using CarbonAware.Aggregators.CarbonAware;
+using CarbonAware.Aggregators.Forecast;
+using CarbonAware.Exceptions;
+using CarbonAware.Tools.WattTimeClient;
+using GSF.CarbonAware.Exceptions;
+using GSF.CarbonAware.Models;
+using Microsoft.Extensions.Logging;
+
+namespace GSF.CarbonAware.Handlers;
+
+internal sealed class ForecastHandler : IForecastHandler
+{
+    private readonly IForecastAggregator _aggregator;
+    private readonly ILogger<ForecastHandler> _logger;
+
+    public ForecastHandler(ILogger<ForecastHandler> logger, IForecastAggregator aggregator)
+    {
```

Comment:
Do we need to inject the Logger? `_logger` is not used anywhere else in this class

Reply:
Missed the log, added it in